pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1613,6 +1624,8 @@ const (
 	BackupTypeTable BackupType = "table"
 	// BackupTypeTiFlashReplica represents restoring the tiflash replica removed by a failed restore of the older version BR
 	BackupTypeTiFlashReplica BackupType = "tiflash-replica"
+	// BackupTypeLog represents the backup is a log backup.
+	BackupTypeLog BackupType = "log"
```

Comment:
Define a new concept will be better since the term 'BackupType' here is not exactly a type but more about granularity. Maybe we can use term "BackupMode", the enums would be BackupModeSnapshot and BackupModeLog.

Reply:
ok